modules/plugins/tomcat/src/main/resources/META-INF/rhq-plugin.xml

```diff
@@ -221,7 +226,26 @@
          dataType="trait"
          displayType="summary"
          description="Tomcat server release identifier" />
-
+      
+      <bundle-target>
+	<destination-base-dir name="Install Directory" description="The top directory where the application server is installed. (i.e. the value found in the 'installationPath' connection property)">
+	    <value-context>pluginConfiguration</value-context>
+	    <value-name>catalinaBase</value-name>
+	</destination-base-dir>
+	<destination-base-dir name="Deploy Directory" description="The deployment directory for a server">
+	    <value-context>pluginConfiguration</value-context>
+	    <value-name>deployDir</value-name>
+	</destination-base-dir>
+	<destination-definition name="Patches">
+	    <accepts bundle-type="Tomcat Patch"/>
+	    <description>Use this location to apply patches to the server</description>
+	    <connection>WEBAPPS</connection>
+	    <referenced-configuration>
+		<all of="pluginConfiguration"/>
+	    </referenced-configuration>
+	</destination-definition>
```

Comment:
What is the reason for <destination-definition name="Patches"> ? I am not aware of tomcat being capable of applying patches and also "Tomcat Patch" bundle type is not known to me. Could you please give a hint?

In case you just copy&pasted this stuff from AS7 plugin, please remove it.